flo-runner/src/main/resources/reference.conf

```diff
@@ -11,3 +11,5 @@ styx.workflow.parameter = "UNKNOWN_PARAMETER"
 styx.workflow.parameter = ${?STYX_PARAMETER}
 styx.workflow.execution = "UNKNOWN_EXECUTION"
 styx.workflow.execution = ${?STYX_EXECUTION_ID}
```

Comment:
Heh, didn't recall we have this already. So we might just make our life easier by removing the internal code and reading environment variables.